chalk-solve/src/rust_ir.rs

```diff
@@ -645,6 +645,8 @@ pub struct OpaqueTyDatum<I: Interner> {
 pub struct OpaqueTyDatumBound<I: Interner> {
     /// Trait bounds for the opaque type.
```

Comment:
Pre-existing, but can we clarify this by adding: "These are bounds that the hidden type must meet."